### PR TITLE
Packages.yaml modified to accept tensorflow as external package

### PR DESCRIPTION
**Pull request by nvarini**

### CODE REVIEW

**Review by alalazo — CHANGES_REQUESTED** [review]
You added the lines to the wrong `packages.yaml`. The one you want to modify is in the `configuration` folder. Can you please update the PR?

File: configuration/packages.yaml [hunk 1]
```diff
@@ -78,6 +78,6 @@ packages:
     buildable: False
     version: ['@0.10.0rc0']
     paths:
-      tensorflow+cuda@0.10.0rc0%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu 
+      tensorflow@0.10.0rc0+cuda@7.5.18+cudnn@4.0.cuda7%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu
```
Comment (alalazo) [inline configuration/packages.yaml]: Did you try the external package before committing? I think you want either to say:
```
tensorflow@0.10.0rc0 ^cuda@7.5.18 ^cudnn@4.0.cuda7%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu
```
or more likely:
```
tensorflow@0.10.0rc0+cuda%gcc@5.3.0: /ssoft/spack/external/tensorflow/0.10.0rc0/tensorflow_venv_gpu
```
When you ask **for variants** you don't need to have **versions** for them.

Comment (nvarini) [reply configuration/packages.yaml]: Well, this version of tensorflow has been compiled with gcc4.9.3 cuda7.5.18 cudnn4.0.cuda7. If you don't have this constraint it does not work

Comment (alalazo) [reply configuration/packages.yaml]: What I mean is that you have a syntax error above.

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
These changes are fine: I'll merge them. We also need an entry in the upper `packages.yaml` to tell Jenkins where do we want this to be installed. And maybe something in `modules.yaml` if we need custom content in the module file.

**Pull request merged**